### PR TITLE
Add basic configuration and info files

### LINKED ISSUE #4: Add basic configuration and info files
* [x] `README.md`, stating your project title, styled with Markdown
* [x] `LICENSE` file. Find template for it in plain text or Markdown format. You may use MIT licence for simplicity
* [x] `.gitattributes`. Find some template for your repo on github
* [x] `.gitignore`. Find typical template for your language (Python, Vim, Linux etc.) @ gitignore.io

You'll need to push a new branch (e.g. `features/3-initial-config`) with atomic commits and create a PR referencing this issue.

### PR DESCRIPTION
**Pull request by PetrushynskyiOleksii**
Resolves #4

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,9 @@
+MIT License
+
+Copyright (c) 2018 Fullname
```
Comment (webknjaz) [inline LICENSE]: Fullname? Is that how you call yourself?

**Review by webknjaz — CHANGES_REQUESTED** [review]
Why did you use that wording in commit messages? I can see that you create files from commit metadata, you shouldn't repeat obvious things in commit message. It's an implementation detail. Your goal is to inform history reader about your intention, of what is the effect of the commit on the project, what you are trying to achieve.

**Review by webknjaz — CHANGES_REQUESTED** [review]
Please refer the issue you are solving in the PR description. (ref. https://help.github.com/articles/closing-issues-using-keywords/)

**Comment by PetrushynskyiOleksii** [comment]
@webknjaz done.

**Review by webknjaz — APPROVED** [review]

**Pull request merged**